### PR TITLE
Rootstock Bridge

### PR DESCRIPTION
**Pull request by jonathansmirnoff**
In this PR we add the Rootstock Bridge. For more information about it go to https://dev.rootstock.io/rsk/architecture/powpeg/
The important thing regarding the integration is that we filter `peg-in` and `peg-out` events.  In the case of `peg-out`, we have to filter different events to grab correctly all the `peg-out` events.
We did several internal tests and the script works well. We decided to start from block 3,616,337 with timestamp 1629390162.

To process the bridge data we have to have a node with logs enabled, so we would like to discuss how have you solved this for other chains.

### CODE REVIEW

File: src/adapters/rootstock/index.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    abi: [
+      "event pegin_btc(address indexed receiver, bytes32 indexed btcTxHash, int256 amount, int256 protocolVersion)",
+    ],
+    logKeys: {
+      blockNumber: "blockNumber",
+      txHash: "transactionHash",
+    },
+    fixedEventData: {
+      from: bridge,
+      token: "RBTC",
```
Comment (jonathansmirnoff) [inline src/adapters/rootstock/index.ts:21]: This is something that we want to double check. The native token of Rootstock is RBTC ( 1 RBTC = 1 BTC).

**Comment by vrtnd** [comment]
need rpc url with logs support to test your adapter.
u can test output txs and volumes using https://github.com/DefiLlama/bridges-server?tab=readme-ov-file#docker
just add `RSK_RPC = some url` with logs support to .env

**Comment by jonathansmirnoff** [comment]
@vrtnd update the PR with your suggestions. Please start testing the adapter from `1629390162` timestamp (or block 3,616,337).
Example: `npm run adapter 1629390162 1706795309 rootstock `

**Comment by vrtnd** [comment]
made some fixes and merged https://github.com/DefiLlama/bridges-server/pull/122

**Pull request closed without merging**